### PR TITLE
Add DataAPI.unwrap with default definition

### PR DESCRIPTION
**Pull request by quinnj**
As discussed in
https://github.com/JuliaData/CategoricalArrays.jl/issues/142.

### CODE REVIEW

**Comment by bkamins** [comment]
Thank you. Could you also bump a version of DataAPI.jl so that we could register it after merging? Thank you.

**Review by bkamins — APPROVED** [review]

File: test/runtests.jl [hunk 1]
```diff
@@ -85,4 +85,9 @@ end
 
 end
 
+@testset "unwrap" begin
+    @test DataAPI.unwrap(1) === 1
+    @test DataAPI.unwrap(missing) === missing
```
Comment (nalimilan) [inline test/runtests.jl:90]: Ah, clever: with that fallback we don't need to use `passmissing(unwrap)`!

**Review by nalimilan — APPROVED** [review]
Thanks!

File: test/runtests.jl (hunk 1 above)
Comment (bkamins) [reply test/runtests.jl:90]: Yes!

**Pull request merged**